src/commands/exit.py

```diff
@@ -0,0 +1,17 @@
+from src.shell import Shell
+from src.utils import Utils
+
+class Exit:
+
+    @staticmethod
+    def do(shell: Shell, line: str):
+        Utils.print('You\'re quitting cryptext')
+        exit(0)
```

Comment:
Here it may be better to call something like `shell.exit()` etc. This would be nicer and would allow all layers of the program to close gracefully if needed (later on for example the shell may want to do something before exiting).